docs/excel/excel-add-ins-conditional-formatting.md

```diff
@@ -0,0 +1,305 @@
+presetFormat.priority = 1;
+
+// Set negative numbers to blue font with green background and set priority 0.
+const cellValueFormat = temperatureDataRange.conditionalFormats
+    .add(Excel.ConditionalFormatType.cellValue);
+cellValueFormat.cellValue.format.font.color = "blue";
+cellValueFormat.cellValue.format.fill.color = "lightgreen";
+cellValueFormat.cellValue.rule = { formula1: "=0", operator: "LessThan" };
+cellValueFormat.priority = 0;
+
+await context.sync();
+
+```
+
+### Mutually exclusive conditional formats
+
+The `stopIfTrue` property of `ConditionalFormat` prevents lower priority conditional formats from being applied to the range. When a range matching the conditional format with `stopIfTrue === true` is applied, no subsequent conditional formats are applied, even if their formatting details are not contradictory.
+
+The following shows two conditional formats being added to a range. Negative numbers will have a blue font with a light green background, regardless of whether the other format condition is true.
+
+![A range with low numbers bolded and in red, unless they are negative, in which case they are not bolded, blue, and have a green background.](../images/excel-conditional-format-stopiftrue.png)
+
+```typescript
+const sheet = context.workbook.worksheets.getItem("Sample");
+queueCommandsToClearAllConditionalFormats(sheet);
+
+const temperatureDataRange = sheet.tables.getItem("TemperatureTable").getDataBodyRange();
+
+// Set low numbers to bold, dark red font and assign priority 1.
+const presetFormat = temperatureDataRange.conditionalFormats
+    .add(Excel.ConditionalFormatType.presetCriteria);
+presetFormat.preset.format.font.color = "red";
+presetFormat.preset.format.font.bold = true;
+presetFormat.preset.rule = { criterion: Excel.ConditionalFormatPresetCriterion.oneStdDevBelowAverage };
+presetFormat.priority = 1;
+
+// Set negative numbers to blue font with green background and 
+// set priority 0, but set stopIfTrue to true, so none of the 
+// formatting of the conditional format with the higher priority
+// value will apply, not even the bolding of the font.
+const cellValueFormat = temperatureDataRange.conditionalFormats
+    .add(Excel.ConditionalFormatType.cellValue);
+cellValueFormat.cellValue.format.font.color = "blue";
+cellValueFormat.cellValue.format.fill.color = "lightgreen";
+cellValueFormat.cellValue.rule = { formula1: "=0", operator: "LessThan" };
+cellValueFormat.priority = 0;
+cellValueFormat.stopIfTrue = true;
+
+await context.sync();
+```
+
+## See Also
+-	[Fundamental programming concepts with the Excel JavaScript API]( https://docs.microsoft.com/office/dev/add-ins/excel/excel-add-ins-core-concepts)
+-	[Work with ranges using the Excel JavaScript API](https://docs.microsoft.com/office/dev/add-ins/excel/excel-add-ins-ranges)
+-	[ConditionalFormat Object (JavaScript API for Excel)]( https://docs.microsoft.com/javascript/api/excel/excel.conditionalformat)
```

Comment:
Maybe also include the two links you mention in the intro, for other articles that contain info about conditional formatting?